### DISCUSSION TITLE
How to use Multi-Agent RL algorithms with skrl

### DISCUSSION THREAD

**Discussion by wangning1996chris** [Toni-SM/skrl#27] [General]
Hi,

Thank you for this great work

We are trying to use MAPPO in conjunction with Omniverse ISAAC GYM.

I would like to ask whether there is a way to extend Single-agent algorithm to Multi-agent algorithm. Do you have any recommended method of doing this?

**Comment by Toni-SM** [comment]
Hi @wangning1996chris 

With the current **skrl** version is not possible to do multi-agent RL, although it would not be difficult to extend the current algorithms to support this type of implementation given the modularity of the library.

The idea is to add support for multi-agent RL agents for the next planned release (v0.9.0)...

Meanwhile, you can try some existing community implementations such as the indicated in [Awesome NVIDIA Isaac Gym](https://github.com/wangcongrobot/awesome-isaac-gym) repo under the *Related GitHub Repos* section, for example, the Safe Multi-Agent Isaac Gym Benchmark
